sdk_container/src/third_party/coreos-overlay/coreos-base/coreos-init/coreos-init-9999.ebuild

```diff
@@ -10,7 +10,7 @@ CROS_WORKON_REPO="https://github.com"
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="235b6141a0b6eb4c1e4dc7778a896f22756898a3" # flatcar-master
+	CROS_WORKON_COMMIT="e0d97d0cb8925320e21541029b085ea65c34910f" # flatcar-master
```

Comment:
It should be probably `2815942c231c6556356eea59a8c15d9685338a06`.

Reply:
Never mind. There is a separate PR for that. https://github.com/flatcar/scripts/pull/1130